dataprofiler/profilers/profile_builder.py

```diff
@@ -658,6 +670,9 @@ def tqdm(l):
                     print("Processing Column {}/{}".format(i+1, len(l)))
                     yield e
 
+        if not sample_size:
+            sample_size = self._get_sample_size(df)
+
```

Comment:
I think this should be called outside the function and passed in

Reply:
moved